src/app/components/normal-component/chat/chat.component.spec.ts

```diff
@@ -252,6 +254,22 @@ describe('ChatComponent', () => {
             const chatDiv: DebugElement = testUtils.findElement('#chatDiv');
             expect(component.scrollTo).toHaveBeenCalledWith(chatDiv.nativeElement.scrollHeight);
         }));
+        it('should not loadChatContent each time user is updated twice', fakeAsync(() => {
+            // Given a chat component, hence subscribed to the current user
+            const userDAO: UserDAO = TestBed.inject(UserDAO);
+            void userDAO.set(UserMocks.CREATOR_MINIMAL_USER.id, UserMocks.CREATOR);
+            tick();
+            ConnectedUserServiceMock.setUser(UserMocks.CREATOR_AUTH_USER, true);
+            testUtils.detectChanges();
+
+            // When the user update to change its last_changed time
+            spyOn(component, 'loadChatContent').and.callThrough();
+            ConnectedUserServiceMock.setUser(UserMocks.CREATOR_AUTH_USER, true);
+            tick();
+
+            // Then loadChatContent should not have been called again
+            expect(component.loadChatContent).not.toHaveBeenCalled();
```

Comment:
pas spécialement "again", just "should not have been called" d'après le code du test

Reply:
well, il a été appellé plus tôt non ?

Reply:
avant le when, donc "when ..., then ... is not called" est complet non ?

Reply:
c'est complet mais pourquoi ne pas ajouter du contexte, c'est jamais une info perdue !
Mais je l'ai enlevé.